src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -310,14 +310,6 @@
     --#{$menu-toggle}__toggle-icon--Color: var(--#{$menu-toggle}--expanded__toggle-icon--Color);
   }
 
-  &:is(:active, .pf-m-active) {
```

Comment:
I think all the`--active--` variables can be removed then since they aren't used? And the `--m-primary--active` and `--m-secondary--active` then as well.